template/Makefile

```diff
@@ -56,6 +56,14 @@ doc: ## Generate odoc documentation
 format: ## Format the codebase with ocamlformat
 	opam exec -- dune build --root . --auto-promote @fmt
 
+.PHONY: assets
+assets: ## Build assets from resources into public
+	npm run build
```

Comment:
What do you think about adding this line to `make build` too?

Reply:
And `make dev`

Reply:
I added `make build` but not `make dev` since parcel has its own watch process for now.

Reply:
If you run both the ocaml watcher and parcel watcher and the ocaml build finishes first, the parcel build will trigger a rebuild of the ocaml build (because `public` is watched by `make dev`). This might be a little annoying, no?

Reply:
Yes, that is annoying, but Sihl needs to be notified about new assets to refresh the browser in the future anyway. I don't have the resources at the moment to properly take care of forking processes and wiring stdout/stderr correctly to implement the "single command" workflow. I didn't find an equivalent for https://www.npmjs.com/package/concurrently which would make this trivial.

Reply:
Ok, I see, in that case => Merging